index.html

```diff
@@ -10,10 +10,98 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
+    
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    
+    <main>
+        <header class="header">
+            <img class="fav_ico" src="img/karma-logo.svg"/>
+            
+            <div>          
+                <nav class="nav_ul">
+                    <a href="#">Meet Karma</a>
+                    <a href="#">How it Works</a>
+                    <a href="#">Store</a>
+                    <a href="#">Blog</a>
+                    <a href="#">Help</a>
+                    <a href="#">Login</a>
+                </nav>
```

Comment:
A nice accessibility pattern for navigation menus is to use a list within the `<nav>` element:

```
<nav>
```suggestion
                <nav class="nav_ul">
                    <ul>
                      <li><a href="#">Meet Karma</a></li>
                      <li><a href="#">How it Works</a></li>
                      <li><a href="#">Store</a></li>
                      <li><a href="#">Blog</a></li>
                      <li><a href="#">Help</a></li>
                      <li><a href="#">Login</a></li>
                    </ul>
                </nav>
```

you can add rules in your CSS file to remove the default browser "bullet point" styles like this:

```css
.nav_ul ul {
  margin: 0;
  padding: 0;
  list-style: none;
}
```